presto-main/src/main/java/io/prestosql/sql/planner/LiteralEncoder.java

```diff
@@ -24,13 +24,16 @@
 import io.prestosql.metadata.ResolvedFunction;
 import io.prestosql.operator.scalar.VarbinaryFunctions;
 import io.prestosql.operator.scalar.timestamp.TimestampToVarcharCast;
+import io.prestosql.operator.scalar.timestamptz.TimestampWithTimeZoneToVarcharCast;
```

Comment:
typo in `Rpresent TIMESTAMP WITH TIME ZONE constants readably in EXPLAIN` commit message